src/main/java/org/kohsuke/github/GHIssueStateReason.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2011, Eric Maupin
```

Comment:
This is off. But I'm not sure if we are encouraged to put our name here, these days. @bitwiseman would know.

Reply:
Looked at a couple recent commits and they don't have any license headers so removed this.